cmdutil/debug/debug.go

```diff
@@ -31,25 +33,72 @@ import (
 // Connect to the debug server with gops:
 //
 //	gops stack localhost:PORT
-func New(l logrus.FieldLogger, port int) *Server {
-	return &Server{
+func New(l logrus.FieldLogger, config Config) *Server {
+	server := &Server{
 		logger: l,
-		addr:   fmt.Sprintf("127.0.0.1:%d", port),
+		addr:   fmt.Sprintf("127.0.0.1:%d", config.Port),
 		done:   make(chan struct{}),
 	}
+	if config.Enabled {
+		server.pprof = NewPProfServer(l, &config.PProf)
+	}
```

Comment:
We're duplicating a server here.

Reply:
@ypaq Are you suggesting that we can do it in this way?
https://github.com/heroku/x/pull/218 ?